src/docs/v10-vs-v11-retrospective.md

```diff
@@ -0,0 +1,643 @@
+### Slots added to `Database`
+
+* `calibration_set`
+* `chemical_entity_set`
+* `chromatographic_category`
+* `configuration_set`
+* `data_generation_set`
+* `instrument_set`
+* `material_processing_set`
+* `protocol_execution_set`
+* `storage_process_set`
+* `workflow_execution_set`
+
+So now, instances of `MetaproteomicsAnalysis`, `MetatranscriptomeAnnotation` and `NomAnalysis` are all collected (or
+aggregated) in `workflow_execution_set`.
+
+These polymorphic collection are enabled by a significant modeling change in v11 of the schema: a strict requirement
+that, within any representation of schema-compliant data, that all instances of all classes declare their own `type`.
+Furthermore, the values in the `type` slot must be the `class_uri` of the instantiated class. That, in turn, means that
+all classes must now declare a `class_uri` and that the `type` slots must be associated with each class. This is
+enforced with the following Python tests:
+
+* test_all_classes_assert_a_class_uri.py
+* test_all_classes_can_use_type_slot.py
+
+Note that classes should not re-associate themselves with any slot that they inherit form a superclass. For example, the
+class definition for `Study`. This no-reasserting rule is tested with `test_inherited_slots_not_reiterated.py`
+
+Also note that the legacy v10 schema did include a `type` slot, but it was used very inconsistently. v10 also ahd a
+similar `designated_class` slot which ahs been retired.
+
+The following slots were removed as a consequence of removing the `Reaction` class and beginning the process of
+deprecating the `Pathway` class:
+
+* `chemical`
+* `direction`
+* `has_part`
+* `has_participants`
+* `is_balanced`
+* `is_diastereoselective`
+* `is_fully_characterized`
+* `is_stereo`
+* `is_transport`
+* `left_participants`
+* `right_participants`
+* `smarts_string`
+* `stoichiometry`
+
+`compound` and `has_solution_components` were removed as a consequence of removing the `Solution` class.
+
+The string-typed `instrument_name` and `used` were removed due to refactoring around the new `Instrument` class and the
+`instrument_used` slot, which is associated with numerous `PlannedProcess` subclasses.
+
+The following have been removed from the `Extraction` class. _more details required_
+
+* `extractant`
+* `extraction_method`
+* `extraction_target`
+
+The string-typed `relevant_protocols` slot has been replaced with the `protocol_link` slot, which uses the `Protocol`
+class as its range.
+
+The string-typed `has_raw_value` has been replaced with `analyte_category`, which has an enumerated range. The following
+are the **permissible values** for `analyte_category`:
+
+* `metagenome`
+* `metatranscriptome`
+* `metaproteome`
+* `metabolome`
+* `lipidome`
+* `nom`
+
+In keeping with the focus on identification rather that quantification, `has_metabolite_quantifications` has been
+replaced with `has_metabolite_identifications` and `metabolite_quantified` has been replaced with
+`metabolite_identified`
```

Comment:
```suggestion
`metabolite_identified`.
```

End sentence with period (there were other instances of this above that I did not comment on).